test/sim/abilities/terashell.js

```diff
@@ -10,56 +10,56 @@ describe('Tera Shell', function () {
 
 	it(`should take not very effective damage when it is at full health`, function () {
 		battle = common.createBattle([
-			[{species: 'Snorlax', ability: 'terashell', moves: ['sleeptalk']}],
+			[{species: 'Terapagos-Terastal', ability: 'terashell', moves: ['sleeptalk']}],
 			[{species: 'Wynaut', moves: ['wickedblow']}],
 		]);
 
 		battle.makeChoices();
-		const snorlax = battle.p1.active[0];
-		const damage = snorlax.maxhp - snorlax.hp;
+		const terapagos = battle.p1.active[0];
+		const damage = terapagos.maxhp - terapagos.hp;
 		assert.bounded(damage, [20, 24], `Tera Shell should yield not very effective damage roll, actual damage taken is ${damage}`);
 
 		battle.makeChoices();
-		assert.bounded(snorlax.maxhp - snorlax.hp - damage, [40, 48], `Tera Shell should not reduce damage, because Snorlax was not at full HP`);
+		assert.bounded(terapagos.maxhp - terapagos.hp - damage, [40, 48], `Tera Shell should not reduce damage, because Snorlax was not at full HP`);
```

Comment:
```suggestion
		assert.bounded(terapagos.maxhp - terapagos.hp - damage, [28, 33], `Tera Shell should not reduce damage, because Terapagos-Terastal was not at full HP`);
```